packages/app/components/stack-modal/StackModal.tsx

```diff
@@ -205,18 +205,19 @@ export const StackModal = ({
             )}
           </div>
         </ModalContent>
-        <ModalFooter>
-          {!stackOrder.cancelledAt && !stackIsComplete(stackOrder) && (
+        {!stackOrder.cancelledAt && !stackIsComplete(stackOrder) ? (
+          <ModalFooter>
             <Button
-              size="sm"
               variant={getConfirmCancelContent().button.action}
               onClick={() => openModal(ModalId.CANCEL_STACK_CONFIRM)}
               width="full"
             >
               {getConfirmCancelContent().button.text}
             </Button>
-          )}
-        </ModalFooter>
+          </ModalFooter>
+        ) : (
+          <div className="pb-6"></div>
```

Comment:
Shouldn't we apply a style to modal instead of adding an empty div?

Reply:
I'd rather keep the short-circuit logic and add a conditional styling (`pb-6`) to the content rather than having an empty div TBH.

Something like abstracting `!stackOrder.cancelledAt && !stackIsComplete(stackOrder)` into a variable and use the same variable for the short-circuit AND the conditional styling for the padding

Reply:
Did extract it into a variable, `stackNotCancelledAndNotComplete`